mobilizon_reshare/config/config.py

```diff
@@ -1,27 +1,85 @@
+import importlib.resources
 import os
-from typing import List
+from pathlib import Path
+from typing import Optional
 
+from appdirs import AppDirs
 from dynaconf import Dynaconf, Validator
 
+import mobilizon_reshare
 from mobilizon_reshare.config import strategies, publishers, notifiers
 from mobilizon_reshare.config.notifiers import notifier_names
 
 from mobilizon_reshare.config.publishers import publisher_names
 
+base_validators = (
+    [Validator("selection.strategy", must_exist=True, is_type_of=str)]
+    + [
+        # strategy to decide events to publish
+        Validator(
+            "publishing.window.begin",
+            must_exist=True,
+            is_type_of=int,
+            gte=0,
+            lte=24,
+        ),
+        Validator(
+            "publishing.window.end", must_exist=True, is_type_of=int, gte=0, lte=24
+        ),
+        # url of the main Mobilizon instance to download events from
+        Validator("source.mobilizon.url", must_exist=True, is_type_of=str),
+        Validator("source.mobilizon.group", must_exist=True, is_type_of=str),
+    ]
+    + [
```

Comment:
You're now loading the preliminary and base validators together. Before only the preliminary validators were loaded and this might be the problem with the test you mentioned. Nonetheless this version is fine.

Reply:
I'm not really sure, I've had some problema with reproducing the expected exception because we loaded also the bundled settings.toml. Also i did this quite late at night, so somthing could've slipped out. I'll look into it